src/Orbit.js

```diff
@@ -152,7 +152,8 @@ class Orbit {
     logger.debug(`Send message to #${channel}: ${message}`)
 
     let data = {
-      content: message.substring(0, 2048),
+      // TODO: Question for Haad, why are you truncating messages?
+      content: new Buffer(message.substring(0, 2048)),
```

Comment:
@haadcode you were truncating every message. Is there a reason for that in Orbit?

Reply:
Haha, yeah! :D That was a quick fix long time ago when some random tester on orbit started flooding with very long messages. It's not a great place to fix that problem but rather should be fixed on the lower levels (orbit-db side). If you can, leave it as it is now (truncate the message, do use buffers if you're changing them) and we'll handle that in the data structures or in orbit-db eventually. Sound good?

Reply:
That makes sense. The message format should be a dag so that it can growth to any length (or at least block large messages at the user interaction level).